src/many/src/server/module/_0_base.rs

```diff
@@ -150,8 +150,7 @@ mod tests {
     use std::sync::{Arc, Mutex};
 
     use crate::{
-        protocol::Attribute, server::module::testutils::call_module,
-        types::identity::cose::tests::generate_random_eddsa_identity,
+        protocol::Attribute, server::module::testutils::call_module, types::identity::cose::testsutils::generate_random_eddsa_identity,
```

Comment:
Why this change? I'm surprised this passes clippy.

Reply:
Needed to change from types::identity::cose::**tests**:: to types::identity::cose::**testsutils**::

It passes `fmt` and `clippy`